Nagstamon/Nagstamon/Server/Zabbix.py

```diff
@@ -261,6 +261,11 @@ def _get_status(self):
                     'triggerid': service['triggerid'],
                 }
 
+                if api_version > '3.0':
```

Comment:
Hmm, good question - I tried to pre-save the same code style as above in the code.

As they switched to 3-digit API version notation (3.0.0 returns now) both ">" and ">=" works, but if we will change version to "3.0.0" in code we should use only ">=".

Honestly, I'm not sure which one is more correct.